packages/node/src/ContentOptionsBuilder.ts

```diff
@@ -62,8 +62,8 @@ class ContentOptionsBuilder {
     return this
   }
 
-  preview() {
-    this.options.preview = true
+  preview(value: boolean) {
```

Comment:
We should set a default value of true, so that `true` doesn't _have_ to be passed in.